tests/FuzzTests.elm

```diff
@@ -8,7 +8,7 @@ import Test exposing (..)
 
 all : Test
 all =
-    describe "List.Zipper"
+    describe "List.Zipper fuzz tests"
```

Comment:
This change allowed me to run `elm-test-rs`: otherwise it complains multiple tests have the same label.